java_console/autotest/src/main/java/com/rusefi/f7nucleo/VssHardwareLoopTest.java

```diff
@@ -0,0 +1,56 @@
+package com.rusefi.f7nucleo;
+
+import com.rusefi.autotest.ControllerConnectorState;
+import com.rusefi.RusefiTestBase;
+import com.rusefi.Timeouts;
+import com.rusefi.core.Sensor;
+import com.rusefi.core.SensorCentral;
+import com.rusefi.enums.engine_type_e;
+import com.rusefi.functional_tests.EcuTestHelper;
+import org.junit.Test;
+
+import static com.rusefi.binaryprotocol.BinaryProtocol.sleep;
+import static com.rusefi.config.generated.Fields.*;
+
+/**
+ * This test relies on jumpers connecting physical pins on f7-nucleo:
+ * PD1<>PC6
+ * PD2<>PA6
+ */
+public class VssHardwareLoopTest extends RusefiTestBase {
```

Comment:
is this some copy-paste? :( can we please reuse code/share code?

Reply:
yup, now I see what I can do to make the pins used configurable